mods/cnc/rules/structures.yaml

```diff
@@ -757,15 +757,15 @@ OBLI:
 		Range: 8c0
 	Bib:
 		HasMinibib: Yes
-	WithChargeAnimation:
+	WithAttackAnimation:
+		AttackSequence: active
+	PreparingAttackSounds:
+		Sounds: obelpowr.aud
 	Armament:
 		Weapon: Laser
 		LocalOffset: 0,-85,1280
-		FireDelay: 0
-	AttackCharge:
-		ChargeAudio: obelpowr.aud
-		ReloadDelay: 40
-		InitialChargeDelay: 50
+		FireDelay: 40
```

Comment:
Fast units can move quite a way (4-5 cells) past the max range during the charge animation, so this is a pretty bad regression.